ngraph_onnx/onnx_importer/ops_bridge/opset_01.py

```diff
@@ -606,11 +592,27 @@ def ConvTranspose(onnx_node, ng_inputs):  # type: (NodeWrapper, List[NgraphNode]
 
 def Pad(onnx_node, ng_inputs):  # type: (NodeWrapper, List[NgraphNode]) -> NgraphNode
```

Comment:
The `Pad` function now has too many cases (`if opset 1`, `if opset 2`). Please split it up into two functions `opset1.Pad` and `opset2.Pad`

Reply:
Done.